backend_new/prisma/schema.prisma

```diff
@@ -0,0 +1,1025 @@
+model AlternateContact {
+  id               String        @id() @default(dbgenerated("uuid_generate_v4()")) @db.Uuid
+  createdAt        DateTime      @default(now()) @map("created_at") @db.Timestamp(6)
+  updatedAt        DateTime      @updatedAt @map("updated_at") @db.Timestamp(6)
+  type             String?
+  otherType        String?       @map("other_type")
+  firstName        String?       @map("first_name")
+  lastName         String?       @map("last_name")
+  agency           String?
+  phoneNumber      String?       @map("phone_number")
+  emailAddress     String?       @map("email_address")
+  mailingAddressId String?       @unique() @map("mailing_address_id") @db.Uuid
+  address          Address?      @relation(fields: [mailingAddressId], references: [id], onDelete: NoAction, onUpdate: NoAction)
+  applications     Applications?
+
+  @@map("alternate_contact")
+}
+
+model AmiChart {
+  id                 String               @id() @default(dbgenerated("uuid_generate_v4()")) @db.Uuid
+  createdAt          DateTime             @default(now()) @map("created_at") @db.Timestamp(6)
+  updatedAt          DateTime             @updatedAt @map("updated_at") @db.Timestamp(6)
+  items              Json
+  name               String               @db.VarChar
+  jurisdictionId     String               @map("jurisdiction_id") @db.Uuid
+  jurisdictions      Jurisdictions        @relation(fields: [jurisdictionId], references: [id], onDelete: NoAction, onUpdate: NoAction)
+  amiChartItem       AmiChartItem[]
+  unitGroupAmiLevels UnitGroupAmiLevels[]
+  units              Units[]
+
+  @@map("ami_chart")
+}
+
+// Note: [birth_month, birth_day, birth_year] formerly max length 8; [phone_number, phone_number_type] formerly max length 16; 
+//       [first_name, middle_name, last_name] formerly max length 64
+// Note: [first_name, last_name] formerly min length 1
+// Note: [email_address] needs to have lower case enforcement
+model Applicant {
+  id                   String        @id() @default(dbgenerated("uuid_generate_v4()")) @db.Uuid
+  createdAt            DateTime      @default(now()) @map("created_at") @db.Timestamp(6)
+  updatedAt            DateTime      @updatedAt @map("updated_at") @db.Timestamp(6)
+  firstName            String?       @map("first_name")
+  middleName           String?       @map("middle_name")
+  lastName             String?       @map("last_name")
+  birthMonth           String?       @map("birth_month")
+  birthDay             String?       @map("birth_day")
+  birthYear            String?       @map("birth_year")
+  emailAddress         String?       @map("email_address")
+  noEmail              Boolean?      @map("no_email")
+  phoneNumber          String?       @map("phone_number")
+  phoneNumberType      String?       @map("phone_number_type")
+  noPhone              Boolean?      @map("no_phone")
+  workInRegion         YesNoEnum?    @map("work_in_region")
```

Comment:
wondering why this isn't a boolean?

Reply:
yeah this is a bit confusing and we may want to take a pass (later on) to convert these to booleans but as it stands rn

```
  @Column({ type: "text", nullable: true })
  @Expose()
  @IsOptional({ groups: [ValidationsGroupsEnum.default] })
  @IsIn(["yes", "no"], { groups: [ValidationsGroupsEnum.default] })
  workInRegion?: string | null
```
workInRegion is technically stored as a string in the db, with a requirement the string is either "yes", "no", or null

this enum change just makes this easier to understand since we effectively this was an enum before